meshing_plugins/include/meshing_base.h

```diff
@@ -0,0 +1,34 @@
+#ifndef MESHING_BASE_H_
+#define MESHING_BASE_H_
+
+#include <pcl/point_cloud.h>
+#include <pcl/point_types.h>
+#include <pcl/PolygonMesh.h>
+
+namespace meshing_base
```

Comment:
So from my limited knowledge, one typically creates a package that contains ONLY the headers related to the interface of the plugin. In a separate package one creates the implementation. The using package includes only a dependency on the definition package.